api/core/workflow/nodes/document_extractor/node.py

```diff
@@ -107,6 +107,33 @@ def _extract_text(*, file_content: bytes, mime_type: str) -> str:
         raise UnsupportedFileTypeError(f"Unsupported MIME type: {mime_type}")
 
 
+def _extract_text_by_file_extension(*, file_content: bytes, file_extension: str) -> str:
+    """Extract text from a file based on its file extension."""
+    match file_extension:
```

Comment:
Have you tested based on the latest code? The `File.extension` should include the dot.

Reply:
just merge the latest branch and add dot for it